### ISSUE TITLE
About pre-trained model : facebook/wav2vec2-large-xlsr-53 & facebook/wav2vec2-base

### ISSUE THREAD

**Issue by LifaSun** [huggingface/transformers#11260]
Hi,

I am trying to load the pre-trained 'no fine-tuning' model of wav2vec2 to exact some features . The models, 'wav2vec2-base' and 'wav2vec2-large-xlsr-53', are not fine-tuned. Why are these files not exactly the same? 

https://huggingface.co/facebook/wav2vec2-base/tree/main
https://huggingface.co/facebook/wav2vec2-large-xlsr-53/tree/main

 'wav2vec2-base'  can be loaded smoothly, but it doesn't work for 'wav2vec2-large-xlsr-53'
processor = Wav2Vec2Processor.from_pretrained("facebook/wav2vec2-base")
model = Wav2Vec2Model.from_pretrained("facebook/wav2vec2-base").to('cuda')

@patrickvonplaten 
Thank you very much!

**Comment by flozi00** [comment]
The xlsr model has no vocab, you need to build the processor yourself

**Comment by flozi00** [comment]
https://huggingface.co/blog/fine-tune-xlsr-wav2vec2
You could check out this blogpost, there is it explained very well

**Comment by LifaSun** [comment]
@flozi00 Thank you very much! Processor is built. 

Actually, I want to visualize the output of hidden layer in the wav2vec model before fine-funing. It seems the output of wav2vec2-base is normal, but the output of wav2vec2-large-xlsr-53 is not. The results are attached (x axis: time, y axis: hidden units).  

The output of hidden layer using wav2vec2-base
![base](https://user-images.githubusercontent.com/6188893/115206484-3b92aa80-a12d-11eb-95ea-13d109250c17.png)

The output of hidden layer using wav2vec2-large-xlsr-53
![xlsr](https://user-images.githubusercontent.com/6188893/115206466-37668d00-a12d-11eb-9c8f-aa11965b2291.png)

Could you explain it? Thank you!
